apps/omg_watcher/lib/exit_processor/core.ex

```diff
@@ -480,6 +547,17 @@ defmodule OMG.Watcher.ExitProcessor.Core do
       get_ifes_with_competitors(request, state)
       |> Enum.map(fn txbytes -> %Event.NonCanonicalIFE{txbytes: txbytes} end)
 
+    invalid_piggybacks =
+      get_invalid_piggybacks(request, state)
+      |> Enum.map(fn {txbytes, inputs, outputs} ->
+        %Event.InvalidPiggyback{txbytes: txbytes, inputs: inputs, outputs: outputs}
+      end)
+
+    # TODO: late piggybacks are critical
```

Comment:
change to `NOTE: late piggybacks are critical, to be implemented in OMG-408`, this deserves an item in our tracker instead of the `credo` woe

Reply:
done